components/playground/playground.go

```diff
@@ -495,6 +498,15 @@ func (p *Playground) handleScaleOut(w io.Writer, cmd *Command) error {
 			fmt.Fprintln(w, color.GreenString(connectMsg, ss[0], ss[1]))
 		}
 	}
+	if cmd.ComponentID == "tiproxy" {
+		addr := p.tiproxys[len(p.tidbs)-1].Addr()
+		if checkDB(addr, cmd.UpTimeout) {
+			ss := strings.Split(addr, ":")
+			connectMsg := "To connect new added TiProxy: mysql --comments --host %s --port %s -u root -p (no password)"
```

Comment:
```suggestion
			connectMsg := "To connect newly added TiProxy: mysql --comments --host %s --port %s -u root -p (no password)"
```

Side note: With MySQL 8.1.0 an newer `--comments` is no longer needed. However it still works. So let's keep it like that for now.

- MySQL Client 8.0.x: `mysql --comments --host %s --port %s -u root -p`
- MySQL Client 8.1.x: `mysql --host %s --port %s -u root -p`
- MySQL Shell 8.x.x: `mysqlsh --sql mysql://root@%s:%s`

Reply:
Oh, thx for the information.